### PR TITLE
Update BoilerplateProjects.md

### PR DESCRIPTION
**Pull request by 17thSep**
A boilerplate project with an example for mocks and spies

## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Review by 17thSep — COMMENTED** [review]
I would like request you to take a look at this change and approve if its upto the standards if not let me know whats missing

**Review by christian-bromann — COMMENTED** [review]
Can we keep the webdriverio maintained ones at the top?

**Comment by 17thSep** [comment]
Sure, i have made the changes.

File: docs/BoilerplateProjects.md [hunk 1]
```diff
@@ -268,6 +255,19 @@ Boilerplate project to run appium tests for native and mobile browser using moch
     - Contains sample test scenarios in mocha
     - Integrated with eslint and allure reporting
 
+### [webdriverio/Mock-and-Spies](https://github.com/17thSep/MonitorNetworkLogs/)
```
Comment (christian-bromann) [inline docs/BoilerplateProjects.md:258]:
```suggestion
### [17thSep/Mock-and-Spies](https://github.com/17thSep/MonitorNetworkLogs/)
```

File: docs/BoilerplateProjects.md [hunk 2]
```diff
@@ -268,6 +255,19 @@ Boilerplate project to run appium tests for native and mobile browser using moch
...
+
+Boiler plate project for
```
Comment (christian-bromann) [inline docs/BoilerplateProjects.md:260]:
```suggestion
Boiler plate project for:
```

File: docs/BoilerplateProjects.md [hunk 3]
```diff
@@ -268,6 +255,19 @@ Boilerplate project to run appium tests for native and mobile browser using moch
...
 
+### [webdriverio/Mock-and-Spies](https://github.com/17thSep/MonitorNetworkLogs/)
+
+Boiler plate project for 
+
+Capture Network Logs
+Capture all GET/POST calls or a specific REST API
+Assert Request parametes
+Assert Response parameters
+Store all the response in a seperate file
```
Comment (christian-bromann) [inline docs/BoilerplateProjects.md:266]:
```suggestion
- Capture Network Logs
- Capture all GET/POST calls or a specific REST API
- Assert Request parametes
- Assert Response parameters
- Store all the response in a seperate file
```

File: docs/BoilerplateProjects.md [hunk 4]
```diff
@@ -268,6 +255,19 @@ Boilerplate project to run appium tests for native and mobile browser using moch
...
+
+You can watch it created in this link
```
Comment (christian-bromann) [inline docs/BoilerplateProjects.md:268]:
```suggestion
You can watch it created in this link.
```

File: docs/BoilerplateProjects.md [hunk 5]
```diff
@@ -255,6 +255,19 @@ Boilerplate project to run appium tests for native and mobile browser using moch
...
+Boiler plate project for:
+
+- Capture Network Logs
+- Capture all GET/POST calls or a specific REST API
+- Assert Request parametes
+- Assert Response parameters
+- Store all the response in a seperate file
+
+You can watch it created in this link.
+https://www.youtube.com/watch?v=Be9IPyxHmLs
```
Comment (christian-bromann) [inline docs/BoilerplateProjects.md:269]:
```suggestion
You can watch it created [on this video](https://www.youtube.com/watch?v=Be9IPyxHmLs).
```

**Review by christian-bromann — APPROVED** [review]
Thank you 👍

**Comment by 17thSep** [comment]
Hi Christian, I don't see the changes in the boilerplate section. Is there a mistake?

**Pull request merged**